deploy/vagrant/create

```diff
@@ -70,6 +70,13 @@ if [[ "$PROVIDER" == "aws" ]]; then
   fi
 fi
 
+# make sure gce.yml exists
+if [[ "$PROVIDER" == “google” ]]; then
```

Comment:
Looks like google had strange quotation marks, which I'll fix. I'm not sure I need curly braces around PROVIDER as the aws line above doesn't use them.